paths_cli/wizard/tools.py

```diff
@@ -0,0 +1,5 @@
+def a_an(obj):
+    return "an" if obj[0].lower() in "aeiou" else "a"
+
+def yes_no(char):
+    return {'y': True, 'n': False}[char]
```

Comment:
probably want to make this capitalization agnostic 
```suggestion
    return {'y': True, 'n': False}[char.lower()]
```

Reply:
done (slightly differently: also allowing full words 'yes' and 'no' -- and with any capitalization, so `yEs` is allowed)